services/web/serve/static.js

```diff
@@ -3,34 +3,36 @@ const Koa = require('koa');
 const { loggingMiddleware } = require('@bedrockio/instrumentation');
 const koaMount = require('koa-mount');
 const koaBasicAuth = require('koa-basic-auth');
+const { pick } = require('lodash');
+const config = require('@bedrockio/config');
 
 const envMiddleware = require('./middleware/env');
 const assetsMiddleware = require('./middleware/assets');
 const historyMiddleware = require('./middleware/history');
 const templateMiddleware = require('./middleware/template');
 const healthCheckMiddleware = require('./middleware/healthCheck');
-const redirectHttpsMiddleware = require('./middleware/redirectHttps');
+
+const allConfig = config.getAll();
```

Comment:
I'm thinking maybe this whole block just goes into the middleware so that it's not in 2 places... makes it less configurable but I'm thinking now that maybe it's not supposed to be

Reply:
ha i was thinking the same, but didn't want to rock the boat. let me update that.

Reply:
actually it was what I meant when we talked but I think I didn't make it clear